backend/api/routers/moodle.py

```diff
@@ -89,16 +90,38 @@ def create_course_build(
         "instructor_firstname": instructor_firstname,
         "instructor_lastname": instructor_lastname,
         "instructor_email": instructor_email,
-        "school_district": school_district_name,
+        "school_district_name": school_district_name,
         "academic_year": academic_year,
         "academic_year_short": academic_year_short,
         "course_name": course_name,
         "course_shortname": maybe_course_shortname,
-        "creator": current_user["email"],
+        "creator_email": current_user["email"],
         "status": status,
     }
 
 
+@router.get("/moodle/course/build", dependencies=[Depends(verify_user)])
+def get_course_builds(
+    db: Session = Depends(database.get_db),
+    academic_year: str = None,
+    instructor_email: str = None,
+) -> list[GetCourseBuildSettings]:
+    course_build = database.get_course_build(db, academic_year, instructor_email)
```

Comment:
Minor: It might be nice to name these plural sine both are going to be / return lists:

```python
    course_builds = database.get_course_builds(db, academic_year, instructor_email)
```